cirq/ops/common_channels.py

```diff
@@ -229,6 +230,110 @@ def depolarize(p: float) -> DepolarizingChannel:
     return DepolarizingChannel(p)
 
 
+@value.value_equality
+class MultiAsymmetricDepolarizingChannel(gate_features.SingleQubitGate):
```

Comment:
We still have two classes: `AsymmetricDepolarizingChannel` (pre-existing) and `MultiAsymmetricDepolarizingChannel` (new). Instead of adding `MultiAsymmetricDepolarizingChannel` this PR should add the new functionality to `AsymmetricDepolarizingChannel`.